free-2.md

```diff
@@ -1,51 +1,65 @@
-# Free project 2
+# Free project 2: Automated Excel spreadsheet population
 
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve people working in consulting, finance or accounting who want to automate or speed up the process of data retrieval and population of Excel spreadsheets. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+People working in these fields often have to parse large amount of information, often in enormous PDF files. These documents like the 10-K contain data in tables that have to be manually copied and pasted into Excel spreadsheets with relationships and equations between cells systematically entered. This is often quite tedious and repetitive work - suggesting an opportunity for a DSL. With this DSL, certain simple equations could be simplified to commands, and strung together with the fluency of a programming language. Ideally, the use of this DSL would free up time and resources for people to devote on less mind-numbing activities. 
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+It would be able to automate some of this tedious work and improve productivity. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+It could potentially be a very useful DSL. I came across this idea after conversations with peers working in economic consulting and my personal experience in Econ classes like Intro to Accounting. 
 
 ### Domain
 _Describe the project's domain in five words._
 
+Automated Excel spreadsheet population
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+At the most basic level, a user could interact with the language on command-line. Although this is not very ideal as it may not be very user-friendly for non-programmers, but would suffice as a minimum-viable product. This would be the right way to interact with the problem domain because creating a web GUI would take away time from development of the MVP. 
```

Comment:
I think this is a good analysis of the tradeoffs.  Oftentimes I feel like programs are packaged in flashy GUIs which do not contribute much to the overall usefulness of the project.  One idea I just thought of (to help address accessibility to users unfamiliar with a command line) would be to have it be launchable through methods other than command line (such as double clicking an application in windows, for example) which then opens up its own terminal for users to interact with when prompted.  This should make the entire process more seamless for new users, since they do not need to learn the concepts of a "shell" or "commandline", they can just open your app normally and use the language when prompted.  Thoughts? Perhaps I am missing something or misjudging what new users would find intuitive.